### ISSUE TITLE
indicator: XCB fatal error when quitting by status icon.

### ISSUE THREAD

**Issue by debug-ito** [debug-ito/wild-bind#4]
When hitting the menu item "Quit" in the popup menu from the status icon, it sometimes emits the following error message.

```
[xcb] Unknown request in queue while appending request
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
wild-bind-debugito: ../../src/xcb_io.c:151: append_pending_request: アサーション `!xcb_xlib_unknown_req_pending' に失敗しました。
中止 (コアダンプ)
```

It's not critical for WildBind, because it's about to quit anyway.

**Comment by debug-ito** [comment]
[Graphics.X11.Xlib.Misc](https://hackage.haskell.org/package/X11-1.8/docs/Graphics-X11-Xlib-Misc.html) module has `initThreads` function. It returns non-zero if successful. Otherwise, it returns zero.

According to [the manpage of `XInitThreads`](https://linux.die.net/man/3/xinitthreads),

> It is only necessary to call this function if multiple threads might use Xlib concurrently. If all calls to Xlib functions are protected by some other access mechanism (for example, a mutual exclusion lock in a toolkit or through explicit client programming), Xlib thread initialization is not required. It is recommended that single-threaded programs not call this function.

**Comment by debug-ito** [comment]
I inserted call to `XInitThreads` in `withX11Front'`, but it still had the problem.
I also removed `notif_disp` (the X11 Display for the thread for the notification debouncer) and used `disp` for the thread. However, the it still has the problem.

**Comment by debug-ito** [comment]
Probably it's fixed by [wild-bind-indicator-1.0.0.0](https://hackage.haskell.org/package/wild-bind-indicator-1.0.0.0).
